RELEASE_NOTES.md

```diff
@@ -1,5 +1,43 @@
 # Push Jobs Client Release Notes
 
+## 3.0
+
+Versions of Chef Push Jobs 3.0 and later may only be used under the terms of the [Chef EULA](https://www.chef.io/end-user-license-agreement/) or another commercial agreement with Chef Software.
+
+Dependency updates:
+- Ruby 2.6.6
+- Chef Infra Client 15.10.12
+- ohai 15.9.1
+- bzip2 1.0.8
```

Comment:
this misc stuff probably isn't worth noting. I doubt anyone cares about ohai, bzip2, cacerts etc